src/test/scala/com/github/dnvriend/serializer/avro4s/AvroTestSpec.scala

```diff
@@ -126,4 +126,84 @@ class AvroTestSpec extends TestSpec {
   implicit class Base64Ops(base64: Base64) {
     def decode[OUT](implicit ev: Decoder[Base64, OUT]): OUT = ev.decode(base64)
   }
-}
\ No newline at end of file
+
+  object SchemaRegistry {
```

Comment:
I factorized the schemas under this "registry". As many posts say, including Kleppman one, Avro has sense with an external registry that holds the schema. Also to avoid DRY violation.